R/quo.R

```diff
@@ -9,12 +9,12 @@
 #' @return An object of class `quosures`
 #'
 #'
-#' @keywords quo
-#' @family quo
+#' @keywords deprecated
+#' @family deprecated
 #'
 #' @export
 quo_c <- function(...) {
-  deprecate_warn(
+  deprecate_stop(
     "0.10.0",
```

Comment:
The one thing I think we should course-correct right now is when we deprecate on `admiraldev` it should reflect the versions of `admiraldev` or `admiral` ? Here its `admiral`

Reply:
oohhh...this gets interesting.  I think it should be `admiraldev`. We use them in admiral, but if we are using them then the message should be telling us to switch to a function in `admiraldev`!

Reply:
![image](https://github.com/pharmaverse/admiraldev/assets/43018762/e4200ef9-4bcc-4884-b940-eb7a99d0d6bb)

Now that you mention, the default lifecycle messaging actually makes it quite clear its `admiraldev` so I'll swap out all these "0.10.0" for "0.3.0" when the deprecation phase 1 process started